packages/vite/src/streaming/registerGlobals.ts

```diff
@@ -25,6 +25,19 @@ export const registerFwGlobals = () => {
       rwConfig.experimental.streamingSsr &&
       rwConfig.experimental.streamingSsr.enabled,
     RWJS_EXP_RSC: rwConfig.experimental?.rsc?.enabled,
+    RWJS_EXP_SSR_GRAPHQL_ENDPOINT: (() => {
```

Comment:
This is a hack, and not a proper solution. I don't know how we could construct this right now.

Reply:
CORS is a browser only issue, so no :)

Reply:
Oh, right. SE server vs FE browser. Of COuRSe.

Reply:
Yeah things get complex pretty quickly 🤣